web/services/subscriptions.go

```diff
@@ -1,14 +1,28 @@
 package services
 
 import (
+	log "github.com/sirupsen/logrus"
+
+	consulApi "github.com/hashicorp/consul/api"
 	"github.com/trento-project/trento/internal/consul"
 	"github.com/trento-project/trento/internal/subscription"
 )
 
+const (
+	SlesIdentifier string = "SLES_SAP"
+	Premium        string = "Premium"
+	Free           string = "Free"
```

Comment:
nitpicking here but... maybe use a different word; maybe community for the non-premium version?

Reply:
Woh I think you're right, good catch

Reply:
We don't have definitive instructions WRT to the branding here, but there is no particular indicator for the "free" version to be implemented: it's either "premium" or it isn't.

Reply:
We can throw some ideas around and then validate with PO, but you can keep it as it is for now.

Reply:
I had my internal backs and forths already with this hehe
Initially I put `Standard`, but what the hell does this mean? hehe
Later on I tried with a `-` character, but It looked so empty

Maybe I could put `Basic`...

But as this is just a wording thing, we can discuss with Lee, Chrisitna, Diego, etc, and It would be an easy thing to change.
This will go to our internal Review state, so I hope it won't go away without this new discussion